docs/reference/search/search-your-data/retrievers-reranking/semantic-reranking.asciidoc

```diff
@@ -0,0 +1,161 @@
+[[semantic-reranking]]
+=== Semantic reranking
+
+preview::[]
+
+[TIP]
+====
+This overview focuses more on the high-level concepts and use cases for semantic reranking. For full implementation details on how to set up and use semantic reranking in {es}, see the <<retriever,reference documentation>> in the Search API docs.
+====
+
+Rerankers improve the relevance of results from earlier-stage retrieval mechanisms.
+_Semantic_ rerankers use machine learning models to reorder search results based on their semantic similarity to a query.
+
+First-stage retrievers and rankers must be very fast and efficient because they process either the entire corpus, or all matching documents.
+In a multi-stage pipeline, you can progressively use more computationally intensive ranking functions and techniques, as they will operate on smaller result sets at each step.
+This helps avoid query latency degradation and keeps costs manageable.
+
+Semantic reranking requires relatively large and complex machine learning models and operates in real-time in response to queries.
```

Comment:
Maybe here extend this to add "and that's why we use it only on a small top-k, as one the of the last steps in a pipeline". Don't quote me on that 😆